agent_pos:
  x: 0.0
  y: 0.0
  z: 1.0
  roll: 0.0
  pitch: 0.0
  yaw: 0.0